data/gui/window/editor_edit_pbl.cfg

```diff
@@ -0,0 +1,373 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to edit an add-ons _server.pbl
+###
+
+#define ROW INPUT LABEL ID TOOLTIP
+[row]
+
+	[column]
+		border = "all"
+		border_size = 5
+		horizontal_alignment = "left"
+
+		[label]
+			id={ID}+"_label"
+			definition = "default"
+			label = {LABEL}
+		[/label]
+	[/column]
+
+	[column]
+		border = "all"
+		border_size = 5
+		horizontal_grow = true
+
+		[{INPUT}]
+			id = {ID}
+			definition = "default"
+			tooltip = {TOOLTIP}
+		[/{INPUT}]
+	[/column]
+
+[/row]
+#enddef
+
+[window]
+	id = "editor_edit_pbl"
+	description = "PBL editor."
+
+	[resolution]
+		definition = "default"
+
+		automatic_placement = true
+		vertical_placement = "center"
+		horizontal_placement = "center"
+
+		maximum_width = 800
+
+		[linked_group]
+			id = "language"
+			fixed_width = true
+		[/linked_group]
+
+		[linked_group]
+			id = "title"
+			fixed_width = true
+		[/linked_group]
+
+		[linked_group]
+			id = "description"
+			fixed_width = true
+		[/linked_group]
+
+		[tooltip]
+			id = "tooltip"
+		[/tooltip]
+
+		[helptip]
+			id = "tooltip"
+		[/helptip]
+
+		[grid]
+
+			[row]
+
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+
+					[label]
+						id = "title"
+						definition = "title"
+						label = _ "PBL Setup"
```

Comment:
Needs a `po:` hint

Reply:
Or change it to "Publishing Setup" instead of using the technical term.

Reply:
Changed to "Add-on Publishing Setup".